modules/util/rotatingfilewriter/writer.go

```diff
@@ -0,0 +1,216 @@
+// Copyright 2023 The Gitea Authors. All rights reserved.
```

Comment:
Move to `modules/log/rotatingfilewriter`?

Reply:
No, it also works with #24257